Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM golang:1.14.2-alpine AS builder
+
+RUN apk add --update --no-cache ca-certificates bash make gcc musl-dev git openssh wget curl bzr
+
+WORKDIR /go/src/mattermod
+
+COPY go.mod .
+COPY go.sum .
+RUN go mod download
+
+COPY . .
+RUN CGO_ENABLED=0 make build
+
+################
+
+FROM alpine:3.12.0
+
+RUN apk --no-cache add ca-certificates
+
+COPY --from=builder /go/src/mattermod/dist/mattermod /usr/local/bin/
+
+WORKDIR /app
+
+RUN mkdir -p /app/logs && chown -R 1000:1000 /app/logs/
+
+USER 1000
+EXPOSE 8080
+
+ENTRYPOINT ["mattermod"]
```

Comment:
Without configuration is going to fail even though I'm passing the config file:
```
$ docker run --name mattermod mattermost/mattermod:e5bbce0 -config config/config-mattermod.default.json
{"level":"error","msg":"unable to load server config","fields":[{"Key":"error","Type":25,"Integer":0,"String":"","Interface":{}}]}
```

Reply:
The way you used it assumes the `config/config-mattermost.default.json` already exists in the container, which does not. The correct way to do it is to map a volume which contains the config.

```bash
docker run \
    -v /path/to/config-mattermod.default.json:/app/config/config.json \
    --name mattermod \
    mattermost/mattermod:e5bbce0 -config /app/config/config.json
````

Reply:
yep given the complexity of the config file I think this is the way (insert Mandalorian gif here)